js/stock-exchange.js

```diff
@@ -0,0 +1,12 @@
+const form = new SearchForm(document.getElementById("searchForm"));
+const marquee = new Marquee(document.getElementById("marqueeContainer"));
+const searchResults = new SearchResult(document.getElementById("stockList"));
+
+form.inputStockSearch.onsearch = function (event) {
```

Comment:
this is an interesting way to bind the from instance to the searchResults instance, but in an ideal code every class should handle the relevant functionalities to itself, meaning the form class should handle the request for data to the server, and the searchResults class should only parse the data and manipulate the DOM. See the ITC solution (when published) to see that option.
Good work anyway! your solution is working, which is good enough!